maltiverse-feed-elastic-connector.py

```diff
@@ -127,12 +138,25 @@
     basic_auth=(arguments.elastic_username, arguments.elastic_password),
 )
 
+if (
+    es_version := get_elastic_server_major_version(es)
+) not in ELASTIC_SUPPORTED_VERSIONS:
+    print(

```

Comment:
For further iterations would be interesting to move this to log. in order to provide a log level input to the script.

Reply:
:+1: